main.go

```diff
@@ -84,6 +84,11 @@ func main() {
 					Name:  "values",
 					Usage: "additional value files to be merged into the command",
 				},
+				cli.IntFlag{
+					Name:  "max-connections",
```

Comment:
@andytom Hi, thx for the PR!

Sorry for being very nit-picky, but I'd prefer `concurrency` over `max-connections` because:

- Helmfile doesn't explicitly deal with connections
- While it doesn't always match `parallelism`, as goroutine's parallelism is capped at GOMAXPROCS

Would you mind if I contributed the change to `concurrency` after this PR gets merged?

Reply:
Not at all, I can change that in this PR if you want?

Reply:
Yes, I appreciate it!

Reply:
Done.